webapp/src/Controller/Jury/JuryMiscController.php

```diff
@@ -62,7 +62,7 @@ public function __construct(EntityManagerInterface $entityManager, DOMJudgeServi
     public function indexAction(Request $request): Response
     {
         $errors = [];
-        return $this->render('jury/index.html.twig', ['errors' => $errors]);
+        return $this->render('jury/index.html.twig', ['errors' => $errors, 'contest_selected' => !is_null($this->dj->getCurrentContest())]);
```

Comment:
Are we using this?
Also: while we are at it, maybe hide the analytics and scoreboard links on the jury interface? Should be able to check `current_contest` in twig to see if there is one

Reply:
I thought about that and my argumentation to leave it like this was that buttons not being there is very annoying. We could make them gray but UX wise I think people finding out they should press the button [change contest] or not showing the button has the same level of frustration.

So I'm fine with your option, if you indeed prefer it over the redirects.

Reply:
OK fair point, buttons and links moving around is also a mess. Let's keep them